src/askui_runner/modules/core/infrastructure/runner/askui.py

```diff
@@ -74,20 +74,17 @@ def create_jinja_env(self, dir_path: str) -> jinja2.Environment:
 
     def render_templates(self, dir_path: str) -> None:
         jinja_env = self.create_jinja_env(dir_path=dir_path)
-        for root, _, files in os.walk(dir_path):
-            for file in files:
-                if not file.endswith(".jinja"):
-                    continue
-                template_file_path = os.path.relpath(os.path.join(root, file), dir_path)
-                target_file_path = os.path.join(
-                    root, file[:-6]
-                )  # TODO Use jinja literal for -6
-                with create_and_open(target_file_path, "w") as f:
-                    f.write(
-                        jinja_env.get_template(template_file_path).render(
-                            self.config.dict()
-                        )
+        templates = [template for template in jinja_env.list_templates() if template.endswith(".jinja")]
+        for template in templates:
+            target_file_path = os.path.join(
```

Comment:
Does join automatically replace the forward slashes in the template with the backward slashes used in Windows as separators?

Reply:
it looks like windows don't "care" about which slashes was used. 
these are the templates paths in my case

```
<>\Local\Temp\tmpw7ard6cpaskui-runner-\helper/jest.setup.ts
<>\Local\Temp\tmpw7ard6cpaskui-runner-\jest.config.ts
```

Reply:
I think we should care, e.g., because we may run into it when debugging and wonder about it, or because there may be consequences we cannot yet see etc.

Here an idea (also used a constant that can be reused in line 77 and makes this less magical):
```python
os.path.join(dir_path, *template[:-len(TEMPLATE_EXTENSION)].split('/'))
```